### ISSUE TITLE
[Bug]: Sound crashing with sound still spinning with Tracker on

### ISSUE THREAD

**Issue by AymericKing** [ShootMe/FallGuysStats#226]
### What happened?

Since 2 weeks, the game has a chance of crashing with the game music continuing to play. my friends still see me in the game BUT MY game has completely closed with the sound spinning. I tried to try without the FG tracker and no more crashes

### Platform

Epic Games

**Comment by Micdu70** [comment]
I really don't think it's related with the tracker. It's a random bug... And random is random. xD

**Comment by AymericKing** [comment]
> I really don't think it's related with the tracker. It's a random bug... And random is random. xD

I tested by completely closing FG tracker and surprised no more crashes

**Comment by qutrits** [comment]
It might be a good idea to check the health of your OS first.
I don't know if [this program](https://pcmanager-en.microsoft.com/) will help you, but if you try it and still have problems, your best option is to reinstall the OS.

**Comment by AymericKing** [comment]
> It might be a good idea to check the health of your OS first.
> I don't know if [this program](https://pcmanager-en.microsoft.com/) will help you, but if you try it and still have problems, your best option is to reinstall the OS.

I don't think this software I was given could solve the problem because it would come from the tracker that crashes my game (100% CPU apparently). Even with the latest update the problem persists which I never had this problem since I switched to pc in June 2022

**Comment by realrellek** [comment]
I only found this nice software recently (shame on me, I know), but did encounter these crashes as well. So I'd strongly lean towards "bug in Fall Guys".

**Comment by GOGE-G1G2MCYT** [comment]
no, this is a glitch i have encountered as of the beginning of SS4 fame pass 2, and as of right now, have just opened the download page for this mod. 6:45 PM :)

**Comment by AymericKing** [comment]
The problem is back, my game is closing randomly with the music playing but I'm still in the game according to my mates. The only way to access the game again is to close it using the application manager

**Comment by realrellek** [comment]
Yes, but that is a FG problem and completely unrelated to the tracker since I have the same problems, but even without the tracker running.

**Comment by AymericKing** [comment]
> Yes, but that is a FG problem and completely unrelated to the tracker since I have the same problems, but even without the tracker running.

So why am I having these crashes

**Comment by realrellek** [comment]
Are you using Wi-Fi? Try wired connection. I can't quite proof it but I feel like it has to do with your ping to the server. Or with the consistency of the ping. My home server (Europe) seems to be pretty ok, when I try to snipe with some streamers (who use US East), I have a lot of crashes as well on Wi-Fi. LAN cable seems a lot better in this regard. What doesn't matter (for me) is if the tracker is running or not.

What also doesn't matter is trying to contact support. All you get there is "try verifying game files. Try updating drivers. Try updating Windows." (DESPITE even attaching log files, so useless these guys)
